lib/class-wp-theme-json-resolver-gutenberg.php

```diff
@@ -244,7 +244,7 @@ public static function get_theme_data( $deprecated = array(), $options = array()
 				$theme_json_data = static::read_json_file( $theme_json_file );
 				$theme_json_data = static::translate( $theme_json_data, $wp_theme->get( 'TextDomain' ) );
 			} else {
-				$theme_json_data = array();
```

Comment:
An empty array is equivalent to an empty v1 theme.json as far as migrations are concerned. I went through all the code and looked for places like this where we want to start with an empty latest version theme.json instead of v1.